script/cadence/contracts/FlowColdStorageProxy.cdc

```diff
@@ -129,7 +127,6 @@ pub contract FlowColdStorageProxy {
                 panic("Cannot destroy a Vault without transferring the remaining balance")
             }
             destroy self.flowVault
```

Comment:
Still trying to figure out whether/how to deal with the removal of this conditioned destroy function

Reply:
We can use `Burner.Burnable.burnCallback()` to encapsulate the logic currently in destroy, however it won't satisfy the same functional requirement in that there's no way to guarantee that this resource isn't destroyed with a balance in the contained vault. The only thing we can do post-1.0 within this custom destroyer is emit an event that would inform listeners that funds were destroyed